content/applications/hr/payroll/reporting.rst

```diff
@@ -2,218 +2,309 @@
+.. tip::
+   Wherever a :guilabel:`➕ (plus)` icon appears on a pivot table, the information can be further
+   grouped. Click on a :guilabel:`➕ (plus)` icon to reveal the available grouping options.
 
-.. image:: reporting/meal-comparison.png
-   :align: center
-   :alt: Meal voucher
+   Click on a :guilabel:`➖ (minus)` icon anywhere on the pivot table to remove that respective
+   grouping.
+
+It is possible to compare the current :guilabel:`Work entry analysis` report to the previous month
+or the previous year. To view these comparisons, click the :guilabel:`⬇️ (down arrow)` icon in the
+search bar to reveal the various :ref:`filter <payroll/filters>` and grouping options.
 
-To export the data in an XLSX format, click the :guilabel:`Download xlsx` icon. The information will
-be downloaded into an Excel spreadsheet.
+In the section titled :guilabel:`Comparison`, click on either :guilabel:`Current Month: Previous
+Period` or :guilabel:`Current Month: Previous Year`. The report updates and displays the previous
+time period values, as well as the :guilabel:`Variation` between the two.
 
-.. image:: reporting/voucher-download.png
+.. image:: reporting/work-entry-comparison.png
    :align: center
-   :alt: Download an Excel spreadsheet of the data by clicking the download button.
+   :alt: A pivot table comparing the work entries of the current month and the previous month.
 
-The data can be inserted into a spreadsheet by clicking the :guilabel:`Insert in Spreadsheet`
-button. A pop-up appears asking which spreadsheet to place the information in. Select an existing
-spreadsheet or enter the name for a new spreadsheet. Click the :guilabel:`Confirm` button to move to
-a spreadsheet view with all the information populated.
+To export the data in an XLSX format, click the :guilabel:`Download xlsx` icon, represented by a
+:guilabel:`⬇️ (down arrow) above a horizontal bar`, located at the far-right of the available icons.
+The information is then downloaded into a spreadsheet.
 
-Once in the spreadsheet view, click on :guilabel:`File`, then select :guilabel:`Save` to save the
-data, or :guilabel:`Save as Template` to save the settings as a template to use in the future.
+The data can also be inserted into a spreadsheet. Click the :guilabel:`Insert in Spreadsheet` button
+and a :guilabel:`Select a spreadsheet to insert your (type of report)` pop-up window appears, asking
+which spreadsheet to place the information in. Select an existing spreadsheet or dashboard, or
+select a new :guilabel:`Blank spreadsheet`. Click the :guilabel:`Confirm` button to move to a
+spreadsheet view with the report added to it.
 
 .. note::
-   The *Documents* application must be installed in order to use the :guilabel:`Insert in
-   Spreadsheet` option.
+   If the *Documents* app is **not** installed, the :guilabel:`Insert in Spreadsheet` option places
+   the newly created spreadsheet in the *Dashboards* app.
```

Comment:
Suggestion to reference the previous section since this is essentially the same instruction to simplify the doc :)